.buildkite/auditbeat/auditbeat-pipeline.yml

```diff
@@ -1,12 +1,18 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
 
 env:
-  IMAGE_UBUNTU_X86_64: "family/core-ubuntu-2204"
-  IMAGE_UBUNTU_ARM_64: "core-ubuntu-2004-aarch64"
+  IMAGE_UBUNTU_2004_X86_64: "family/core-ubuntu-2004"
```

Comment:
maybe also add `family/core-ubuntu-1804` it is in the support matrix of auditbeat